### PR TITLE
Add to $PATH in the .Net Core Host pkg

### PR DESCRIPTION
**Pull request by Sridhar-MS**
Earlier it was incorrectly done in .Net CLI pkg

cc: @blackdwarf

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/2062)

### CODE REVIEW

**Comment by brthor** [comment]
:shipit:

File: packaging/osx/clisdk/scripts/postinstall [hunk 1]
```diff
@@ -9,10 +9,6 @@ INSTALL_DESTINATION=$2
 
 
 # A temporary fix for the permissions issue(s)
-chmod -R 755 $INSTALL_DESTINATION
-
-# Add the installation bin directory to the system-wide paths
-echo $INSTALL_DESTINATION | tee -a /etc/paths.d/dotnet
-
+chmod -R 755 $INSTALL_DESTINATIONs
```
Comment (mellinoe) [inline packaging/osx/clisdk/scripts/postinstall]: Is this a typo or is there supposed to be an "s" at the end of $INSTALL_DESTINATION?

Comment (Sridhar-MS) [inline packaging/osx/clisdk/scripts/postinstall]: Aargh.. Its a typo.

Comment (blackdwarf) [inline packaging/osx/clisdk/scripts/postinstall]: @Sridhar-MS :)

**Pull request merged**